torch/nn/_intrinsic/qat/modules/conv_fused.py

```diff
@@ -219,6 +227,14 @@ def __init__(self,
     def forward(self, input):
         return self.observer(F.relu(super(ConvBnReLU2d, self)._forward(input)))
 
+    def update_bn_stats(self):
```

Comment:
this module inherits from ConvBn2d, do we still need this function?

Reply:
Good point, have moved this to the base class (convBn2d) and removed it in convbnrelu2d